control-plane/grpc/proto/v1/volume/volume.proto

```diff
@@ -179,6 +179,26 @@ message VolumeState {
   map<string, ReplicaTopology> replica_topology = 5;
   // volume usage information
   optional VolumeUsage usage = 6;
+  // volume health information
+  optional VolumeHealth health = 7;
+}
+
+// Extra volume information.
+message VolumeHealth {
+  // Indicates if the volume target was cleanly shutdown.
+  bool cleanShutdown = 1;
```

Comment:
LGTM, but the word clean might not be very clear, what about consistent or insync?

Reply:
Will merge as is for now, and will re-discuss this later on